api/http/workflow.go

```diff
@@ -120,8 +120,12 @@ func (h WorkflowController) StartWorkflow(c *gin.Context) {
 		return
 	}
 
+	workflowVersion := ""
 	for key, value := range inputParameters {
 		valueSlice, ok := value.([]interface{})
+		if key == "version" {
+			workflowVersion = value.(string)
+		}
```

Comment:
This is quite dangerous, and it makes impossible to pass in a `version` input parameter.
I personally believe this kind of information should be passed in as HTTP header, e.g.
`X-Worfklows-Min-Version: 1`

What do you think?

Reply:
> This is quite dangerous, and it makes impossible to pass in a `version` input parameter.

here with POST the version is passed as a input parameter.

> I personally believe this kind of information should be passed in as HTTP header, e.g. `X-Worfklows-Min-Version: 1`
> 
> What do you think?

no problema! something along [these lines](https://github.com/mendersoftware/workflows/pull/168/commits/0229694b7606c49632ab184a438bcf0f35d3c9f3#diff-cd61fe21b4b29387ea1e98b73e93330fcf54c7a818363030a7c869ecbdd46a1bR128)?